docs/website/docs/dlt-ecosystem/llm-tooling/mcp-server.md

```diff
@@ -0,0 +1,202 @@
+
+- Connect to the destination to retrieve data. This enables table previews and light text-to-SQL capabilities.
+
+### Resources
+
+- LLM-optimized dlt documentation pages. These can be selected and added to your prompt to help the LLM generate valid `dlt` code.
+
+### Prompts
+
+- Tutorial-like instructions that puts the LLM in "assistant mode". For example, the LLM can ask you questions about your data and "think" with you to select the right loading strategy.
+
+- Command-like instructions that gives the LLM a task to complete. For example, the LLM can initialize a new pipeline. This is akin to a conversational command line tool.
+
+
+## Installation
+
+The `dlt` and `dlt+` MCP servers are intended to run locally on your machine and communicate over standard I/O. Typically, the MCP server process is launched by the MCP client, i.e., the IDE. We will use the [uv package manager](https://docs.astral.sh/uv/#installation) to launch the MCP server.
+
+The next sections include client-specific instructions, references, and snippets to configure the MCP server. They are mainly derived from this `uv` command:
+
+```sh
+uv tool run --with "dlt-plus[mcp]==0.9.0" dlt mcp run
+```
+
+To explain each part:
+- [uv tool run](https://docs.astral.sh/uv/guides/tools/) executes the command in an isolated virtual environment
+- the `--with PACKAGE_NAME` specify the Python dependencies required by the MCP server
+- `dlt-plus==0.9.0` pins a specific `dlt-plus` version (where the MCP code lives). We suggest at least pinning the `dlt-plus` version to provide a consistent experience
+- `dlt mcp run` is a CLI command found in `dlt+`
+
+Then, to enable the MCP server and tool usage, several IDEs require you to enable "tool/agent/mcp mode".
+
+### dlt+ MCP server
+
+To run the `dlt+` MCP server, you will need to set your [dlt+ License](../../plus/getting-started/installation#licensing) globally in `~/.dlt/secrets.toml` or in an environment variable (must be set before lauching the IDE).
+
+Then, use `dlt mcp run_plus` to launch the server instead of `dlt mcp run`. If the `dlt+` license is missing, the `dlt` MCP server will be launched instead. You can tell the two apart by the tools, resources, and prompts available­.
+
+
+### Continue
+
+With Continue, you can use [Continue Hub](https://docs.continue.dev/hub/introduction) for a 1-click install of the MCP, or a local config file. Select `Agent Mode` to enable the MCP server.
+
+#### Continue Hub
+See the [dltHub page](https://hub.continue.dev/dlthub) and select the `dlt` or `dlt+` Assistants. This bundles the MCP with additional Continue-specific features. You can also select the `dlt` or `dlt+` MCP blocks to install the server exclusively.
+
+#### Local
+You can define an assistant locally with the same YAML syntax as the Continue Hub by adding files to `$PROJECT_ROOT/.continue/assistants`. This snippet creates an assistant with the MCP only.
+
+```yaml
+# local_dlt.yaml
```

Comment:
```suggestion
```yaml title="local_dlt.yaml" 
```

Reply:
Sorry for misleading suggestion. Looks like we need to update our snippets checker so it parses [code title](https://docusaurus.io/docs/markdown-features/code-blocks#code-title) as well. @sh-rp how hard would it be?

Reply:
closing this for now